pom.xml

```diff
@@ -223,10 +230,6 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
-                        <configuration>
-                            <!-- Skip the unit tests when the it profile is turned on -->
-                            <skip>true</skip>
-                        </configuration>
```

Comment:
Do we want to run unit tests along with integration tests? If yes then we need to update the bamboo builds to reflect the same

Reply:
Yes, that is the norm. It makes sense to me. I can run `mvn verify` to run _just_ unit tests, I can also run `mvn -Pit verify` to run _all_ the tests. If we do not do this you'd have to do two invocations to run all tests, which seems suboptimal.

I don't think we necessarily need to update builds, if we feel that unit tests unnecessarily slows down the integration test run in the build system we can add properties to disable just unit tests.